src/models/instance.ts

```diff
@@ -1,11 +1,16 @@
+///<reference path="../types/modules.d.ts" />
```

Comment:
Triple-slash directive should not be needed now, tsconfig already looks up for .d.ts files on src and it should correctly find the d2/uid module definition without forcing the compiler to look it up. At least I believe it worked fine when I tested it on my end, could you double check?

Reply:
sure